lib/Controllers/create.js

```diff
@@ -17,7 +17,29 @@ Create.prototype.plurality = 'plural';
 Create.prototype.write = function(req, res, context) {
   context.attributes = _.extend(context.attributes, req.body);
 
-  var self = this;
+  var self = this,
```

Comment:
this whole code block is what I was referring to in my previous comment. Does this actually need to be run every time you have a write milestone? Can't you cache this so you only do it once when you create the resource?